packages/dex2jar.vm/tools/chocolateyinstall.ps1

```diff
@@ -0,0 +1,10 @@
+$ErrorActionPreference = 'Stop'
+Import-Module vm.common -Force -DisableNameChecking
+
+$toolName = 'd2j-dex2jar'
```

Comment:
I have found `d2j-dex2jar` useful recently when analysing Android malware. I think dex2jar includes other tools as well that we may want to create a shortcut for. Opinions?